src/main/java/seedu/duke/command/list/ListCommand.java

```diff
@@ -17,45 +17,27 @@ public ListCommand(Parser.TypeOfEntries typeOfEntry) {
         this.typeOfEntry = typeOfEntry;
     }
 
-    /**
-     * Lists all the tasks from the task list.
-     *
-     * @throws TaskManager.TaskNotFoundException If the task is not found in the task list.
-     */
-    private void listTask() throws TaskManager.TaskListEmptyException {
-        TaskManager.list();
-    }
-
-    /**
-     * Lists all the modules from the module list.
-     *
-     * @throws ModuleManager.ModuleNotFoundException If the module is not found in the module list.
-     */
-    private void listModule() throws ModuleManager.ModuleListEmptyException {
-        ModuleManager.list();
-    }
-
     /**
      * Lists the tasks/modules from their respective lists.
      *
      * @return CommandResult containing acknowledgement of the delete.
      */
     @Override
     public CommandResult execute() {
-        try {
-            switch (typeOfEntry) {
-            case TASK:
-                listTask();
-                break;
-            case MODULE:
-                listModule();
-                break;
-            default:
-                break;
-            }
-            return new CommandResult(MESSAGE_LIST_PRINTED);
-        } catch (ModuleManager.ModuleListEmptyException | TaskManager.TaskListEmptyException e) {
+        String output = "";
+        switch (typeOfEntry) {
+        case TASK:
+            output = TaskManager.list();
+            break;
+        case MODULE:
+            output = ModuleManager.list();
+            break;
+        default:
+            break;
+        }
+        if (output.equals("")) {
```

Comment:
Maybe use .isEmpty()?

Reply:
Should be `output == null` because TaskManager/ModuleManager returns `null` if the respective lists are empty.